### ISSUE TITLE
Filter on Security Performance Does Not Filter Transactions

### ISSUE THREAD

**Issue by flywire** [buchen/portfolio#2371]
**Describe the bug**
Performance Calculation report with filter does not filter the transactions.

**To Reproduce**
1. Display Performance Calculation report with filter on Entire Portfolio
2. Select a security held in multiple accounts then display transactions
3. Set filter to only display one of the accounts
4. See error - transactions not filtered by account

**Expected behavior**
Filter should apply to transactions too.

**Screenshots**
![image](https://user-images.githubusercontent.com/11288701/128502415-01dfe182-187f-49d2-9ca7-ec96ccaca489.png)
Filter Entire Portfolio

![image](https://user-images.githubusercontent.com/11288701/128502481-bcd13979-abfb-4ca8-b9fb-10bd109475dc.png)
No change when filtered for Broker Securities/Broker Funds/Bank

Version: 0.54.2 (Jul. 2021)
Platform: win32, x86_64
Java: 11.0.4+11-LTS, Azul Systems, Inc.

**Comment by chirlu** [comment]
The bottom area just gives general information about the security.

**Comment by flywire** [comment]
No, the filter carries through to the bottom section of other reports but not this one.

**Comment by flywire** [comment]
No, the filter carries through to the bottom section of other reports but not this one.

eg Filter works on Income & Expenses, Fees Subreport, Transactions Subreport.

Following reports support filter:
* Statement of Assets: Trades
* Performance Calculation: Trades
* Securities: Trades
* Income & Expense: Seems to filter every top-line subreport

Interestingly, Trades report has no Account filter.

**Comment by flywire** [comment]
I don't know when, but fixed by Version: 0.81.0 (December 2025).
